### PR TITLE
Drawer-like styles for mobile, and an optional invisible layer for closing the datepicker

### PR DESCRIPTION
**Pull request by mmanneva**
We are using ng2-datetime-picker in a project I'm working on and we needed a couple of features that were missing so I added them. 

* Added a transparent, fixed position layer that covers the viewport and closes the datepicker when clicked. I find this a better option than to listen to a global click or to the blur event of the input that opened the datepicker. I added a flag `show-close-layer` to control this.

* Added a slide-in drawer style implementation for the datepicker for mobile viewports (arbitrarily chosen to be max-width: 767px).

* Made the next/prev navigation slightly bigger to be easier to hit with a mouse.

Feel free to use the ideas or the code if you wish!

Cheers,

Manne

### CODE REVIEW

File: src/ng2-datetime-picker.component.ts [hunk 1]
```diff
@@ -408,6 +453,7 @@ export class Ng2DatetimePickerComponent {
   }
 
   public close() {
+    console.log('töttöröö');
```
Comment (allenhwkim) [inline src/ng2-datetime-picker.component.ts]: Please remove or comment out this debug log

File: src/ng2-datetime-picker.directive.ts [hunk 2]
```diff
@@ -263,6 +265,7 @@ export class Ng2DatetimePickerDirective implements OnInit, OnChanges {
   };
 
   hideDatetimePicker = (event?): any => {
+    console.log('what');
```
Comment (allenhwkim) [inline src/ng2-datetime-picker.directive.ts]: Please remove or comment out this debugging log

File: src/ng2-datetime-picker.component.ts (hunk 1 above)
Comment (mmanneva) [inline src/ng2-datetime-picker.component.ts]: Whoops, sorry! That was from when I was trying to find which event was closing the datepicker. :)

File: src/ng2-datetime-picker.directive.ts (hunk 2 above)
Comment (mmanneva) [inline src/ng2-datetime-picker.directive.ts]: Whoops, sorry! That was from when I was trying to find which event was closing the datepicker. :)

**Pull request merged**